src/main/java/bookstore/controller/BookController.java

```diff
@@ -16,39 +22,49 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
```

Comment:
I think it looks better

```suggestion
@Tag(name = "Book store API", description = "Endpoints for managing books")
```